app/components/inline_code_component.html.erb

```diff
@@ -0,0 +1 @@
+<span class="inline-code"><%= @snippet %></span>
```

Comment:
Would this be more semantic as a [`<code>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/code) tag? Given the CSS styling and class name, that seems to be the most appropriate HTML tag to use.

Whereas [`<span>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span) has no semantic value.